docs/pages/en-us/Features.md

```diff
@@ -32,5 +32,8 @@ The service is designed for durability as Elasticsearch provides data redundancy
 
 ## High-Availability
 The service is highly available as Elasticsearch replicates and distributes data across it's scalable, durable cluster.  
+
+## Audit Web Service API and Client
+Audit Web Service for logging application audit events through its RESTful interface which sends them through to Elasticsearch.
```

Comment:
Maybe reword this to big it up as a feature?

"The Audit Web Service is effective for logging application events ... which forwards them through to Elasticsearch"